packages/components/src/navigable-container/container.tsx

```diff
@@ -67,14 +88,14 @@ class NavigableContainer extends Component {
 		return null;
 	}
 
-	getFocusableIndex( focusables, target ) {
-		const directIndex = focusables.indexOf( target );
-		if ( directIndex !== -1 ) {
-			return directIndex;
-		}
+	getFocusableIndex(
+		focusables: Element[],
+		target: Element,
+	) {
+		return focusables.indexOf( target );
```

Comment:
What do you think of this simplification?

I changed this because there was a warning from `getFocusableIndex()` sometimes having no return value.

Instead of checking whether `directIndex !== -1`, this simply returns the `directIndex`.

The only function that calls `getFocusableIndex` already makes sure `this.getFocusableIndex()` isn't `-1`.

So it should be fine to simply return `focusables.indexOf( target );`

https://github.com/WordPress/gutenberg/blob/5ee4511a853485d92d99e94403f2c2a35bfea621/packages/components/src/navigable-container/container.tsx#L84-L88